data-manipulation/hashing/djb2/hash-data-using-djb2.yml

```diff
@@ -7,7 +7,7 @@ rule:
       - still@teamt5.org
     scope: function
     mbc:
-      - Data::Non-Cryptographic Hash [C0030]
+      - Data::Non-Cryptographic Hash [C0030.006]
```

Comment:
Is this ID in MBC yet? I don't see it in https://github.com/MBCProject/mbc-markdown/blob/master/micro-behaviors/data/noncryptographic-hash.md or in the STIX file.

Reply:
C0030.006 is in the mbc.json STIX data now, and it will be in the mbc-markdown shortly.